src/models/rule_store.rs

```diff
@@ -70,18 +70,18 @@ impl RuleStore {
       r.name().eq(&rule.name()) && r.replace().eq(&rule.replace()) && r.query().eq(&rule.query())
     }) {
       #[rustfmt::skip]
-      debug!("{}", format!("Added Global Rule : {:?} - {}", r.name(), r.query()).bright_blue());
+      debug!("{}", format!("Added Global Rule : {:?} - {}", r.name(), r.query().0).bright_blue());
```

Comment:
As mentioned in a different comment, do we want to be referencing `.0` or encapsulate this in a meaningfully named method?